yoyodyne/train.py

```diff
@@ -44,17 +44,28 @@ def _get_callbacks(save_top_k: int, patience: Optional[int] = None) -> List:
     Args:
         save_top_k (int).
         patience (int, optional).
+        save_best (bool). If `True`, save the best checkpoint(s) using the validation metric.
 
     Returns:
         List: callbacks.
     """
-    trainer_callbacks = [
-        callbacks.ModelCheckpoint(
+    if save_best:
+        checkpoint_callback = callbacks.ModelCheckpoint(
             save_top_k=save_top_k,
             monitor="val_accuracy",
             mode="max",
             filename="model-{epoch:03d}-{val_accuracy:.3f}",
-        ),
+        )
+    else:
+        checkpoint_callback = callbacks.ModelCheckpoint(
+            save_top_k=save_top_k,
+            every_n_epochs=1,
+            save_on_train_epoch_end=True,
+            filename="model-{epoch:03d}",
+        )
+
```

Comment:
Nit, but could you eliminate the blank line here? My personal style guide for this project doesn't use them function-internally, since I think they make it hard to chunk functions visually.